### PR TITLE
Updateing the instructions for big mode

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

File: DEVELOPMENT.rst [hunk 1]
```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
...
+2. Install python dependencies::
+
+    $ pip install -r requirements_production.txt
 
 3. Change settings.py
 
-(See comments in the settings)
+  (See comments in the settings)
+
+  The relevant settings are: DATABASES, use_redis, CHANNEL_LAYERS, CACHES
```
Comment (emanuelschuetze) [inline DEVELOPMENT.rst]: only DATABASE and use_redis = True

File: DEVELOPMENT.rst [hunk 2]
```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
...
+
+4. If you created a fresh Database run this command to populate it::
```
Comment (emanuelschuetze) [inline DEVELOPMENT.rst]: database with small "d"

File: DEVELOPMENT.rst [hunk 3]
```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
...
 
 
 Installation OpenSlides in big mode
 ===================================
 
-1. Install PostgreSQL und redis:
+1. Install PostgreSQL and redis
+-------------------------------
+
+    Just apt-get these packets::
```
Comment (normanjaeckel) [inline DEVELOPMENT.rst]: packages

Comment (normanjaeckel) [inline DEVELOPMENT.rst]: Also please dedent this line.

File: DEVELOPMENT.rst [hunk 4]
```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
...
+
+    $ sudo apt-get install postgresql redis-server libpq-dev --yes
+
+2. Configure postgresql
```
Comment (normanjaeckel) [inline DEVELOPMENT.rst]: This part is very complex. I do not remember this being so complex at all. Just installed PostgreSQL and added a database user and a new database.

File: DEVELOPMENT.rst [hunk 5]
```diff
@@ -107,32 +107,91 @@ All other commands are the same as for GNU/Linux and Mac OS X.
...
+      $ python manage.py runworker [--threads 2]
 
-4. Start one or more workers:
+    Set the DJANGO_SETTINGS_MODULE and the PYTHONPATH to start daphne::
 
-python manage.py runworker
+      $ DJANGO_SETTINGS_MODULE=settings PYTHONPATH=personal_data/var/ daphne openslides.asgi:channel_layer
 
-5. Start daphne. Set the DJANGO_SETTINGS_MODULE and the PYTHONPATH
+  b) Geiss: Just run OpenSlides with this extra argument::
```
Comment (normanjaeckel) [inline DEVELOPMENT.rst]: In big mode we should not run geiss via openslides start but in extra process or terminal.

File: DEVELOPMENT.rst (hunk 4 above)
Comment (FinnStutzenstein) [inline DEVELOPMENT.rst]: Hmm.. I didn't use postgresql before and googled for 'configure postgresql'. This is (a little bit updated) version of [this result](http://stackoverflow.com/questions/1471571/how-to-configure-postgresql-for-the-first-time).

If you have a lighter quick setup guide please let me know.

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze @normanjaeckel Please check again.

File: DEVELOPMENT.rst (hunk 4 above)
Comment (normanjaeckel) [inline DEVELOPMENT.rst]: Just installed PostgreSQL via apt-get and then added DB-user with password and DB with the commands given here: https://wiki.ubuntuusers.de/PostgreSQL/#Administration.

**Comment by normanjaeckel** [comment]
I would like to test it myself on Ubuntu 16.04.

**Comment by FinnStutzenstein** [comment]
As of the proposal from @emanuelschuetze : I've split up the requirements_production, so a 'normal' user (just using the sqlite db) does not have to install all deps for postgreSQL as well as not having to apt-get all the requirements he don't need.
If you want to have the big mode, you can now do a `pip install -r requirements_postgresql.txt` for using postgreSQL.

**Comment by emanuelschuetze** [comment]
Ok for me.
@normanjaeckel Please review (,test) and merge.

**Comment by emanuelschuetze** [comment]
@normanjaeckel ping, I prefer to merge this now, that we can make a full test based on master. Later you can create a new pull request with more changes.

**Comment by normanjaeckel** [comment]
You can rebase and merge if you like but I will change the name of the files and the instructions. This work is still in progress.

**Comment by normanjaeckel** [comment]
@emanuelschuetze Please have a new review. Then it is ready to merge.
@FinnStutzenstein Sorry but I accitently flushed your commit away.

File: DEVELOPMENT.rst [hunk 6]
```diff
@@ -104,35 +121,85 @@ All other commands are the same as for GNU/Linux and Mac OS X.
...
+1. Install and configure PostgreSQL and Redis
+---------------------------------------------
+
+Install `PostgreSQL <https://www.postgresql.org/>`_ and `Redis
+<https://redis.io/>`_. For Ubuntu 16.04 e. g. run::
+
+    $ sudo apt-get install postgresql libpq-dev redis-server
+
+Be sure that database and redis server is running. For Ubuntu 16.04 e. g. this
+was done automaticly if you used the package manager.
```
Comment (emanuelschuetze) [inline DEVELOPMENT.rst]: typo: "automatically"

File: DEVELOPMENT.rst [hunk 7]
```diff
@@ -104,35 +121,85 @@ All other commands are the same as for GNU/Linux and Mac OS X.
...
+Then add database user and database. For Ubuntu 16.04 e. g. run::
+
+    $ sudo -u postgres createuser --pwprompt --createdb openslides
+    $ sudo -u postgres createdb --owner=openslides openslides
+
+
+2. Install additional packages
+------------------------------
+
+Install some more Python packages::
```
Comment (emanuelschuetze) [inline DEVELOPMENT.rst]: suggestion: "... some more _required_ Python packages..."

File: DEVELOPMENT.rst (hunk 6 above)
Comment (normanjaeckel) [inline DEVELOPMENT.rst]: Done

File: DEVELOPMENT.rst (hunk 7 above)
Comment (normanjaeckel) [inline DEVELOPMENT.rst]: Done.

**Pull request merged**